paymentsheet/src/test/java/com/stripe/android/paymentsheet/verticalmode/ManageScreenUIScreenshotTest.kt

```diff
@@ -0,0 +1,54 @@
+package com.stripe.android.paymentsheet.verticalmode
+
+import androidx.compose.foundation.layout.padding
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.stripe.android.model.PaymentMethod
+import com.stripe.android.model.PaymentMethodFixtures
+import com.stripe.android.paymentsheet.FakeManageScreenInteractor
+import com.stripe.android.screenshottesting.FontSize
+import com.stripe.android.screenshottesting.PaparazziRule
+import com.stripe.android.screenshottesting.SystemAppearance
+import com.stripe.android.utils.screenshots.PaymentSheetAppearance
+import org.junit.Rule
+import org.junit.Test
+
+internal class ManageScreenUIScreenshotTest {
+
+    @get:Rule
+    val paparazziRule = PaparazziRule(
+        SystemAppearance.entries,
```

Comment:
For layout based UI, we typically just use the default appearance, since the individual rows should already test each of the variants.

Reply:
thanks! Making that update now